lms/views/lti/basic_launch.py

```diff
@@ -243,65 +201,55 @@ def unconfigured_basic_lti_launch_not_authorized(self):
         """
         return {}
 
-    @view_config(
-        authorized_to_configure_assignments=True,
-        decorator=[],  # Disable the default decorators just for this view.
-        route_name="module_item_configurations",
-        schema=ConfigureModuleItemSchema,
-    )
-    def configure_module_item(self):
+    @classmethod
+    def _report_to_h(cls, context, request):
         """
-        Respond to a configure module item request.
+        Update H with all the required information.
 
-        This happens after an unconfigured assignment launch. We show the user
-        our document selection form instead of launching the assignment, and
-        when the user chooses a document and submits the form this is the view
-        that receives that form submission.
-
-        We save the chosen document in the DB so that subsequent launches of
-        this same assignment will be DB-configured rather than unconfigured.
-        And we also send back the assignment launch page, passing the chosen
-        URL to Via, as the direct response to the content item form submission.
+        TODO! - What is this really up to? All sorts. Think of a better name
```

Comment:
I'm not sure how to summarise what this is doing, part of it is reporting to H, part is just local stuff. Suggestions very welcome? Maybe two functions?